### ISSUE TITLE
Navbar links are permanently displayed on smaller screen sizes. 

### ISSUE THREAD

**Issue by tomzacchia** [systers/communities#115]
## Description
As a systers community member,
I need navigate to different parts of the website,
so that I can view content of interest.

## Mocks

![screen shot 2018-09-04 at 4 44 50 pm](https://user-images.githubusercontent.com/26035225/45056912-35c22680-b062-11e8-87f7-ca64e450f733.png)
![screen shot 2018-09-04 at 4 44 45 pm](https://user-images.githubusercontent.com/26035225/45056898-2d69eb80-b062-11e8-9b82-ac3b8d19397b.png)

### Update [Required]
- [ ] Links are not displayed on smaller screen sizes unless toggled by hamburger button
- [ ] nav-bar.component.html

## Definition of Done
- [ ] All of the required items are completed.
- [ ] Approval by 1 mentor.

## Estimation
1 hour

**Comment by Tharangi** [comment]
@tomzacchia could you explain the issue? 

I can't find any issue/ any not user-friendly behavior in the navigation bar responsiveness.

**Comment by tomzacchia** [comment]
@Tharangi. I am viewing https://systers-communities.herokuapp.com/ on Chrome and at breakpoint 1200px (width) the navbar links collapse and are replaced by the hamburger icon. After this transition the navbar links can be seen hidden behind the div with class "carousel-inner", if you look at the top left corner of the carousel image you can see the first two letters of "Communities, Calendar and Contact Us"

Hope this helps.

**Comment by Tharangi** [comment]
Sorry!, I still couldn't recreate it.
![image](https://user-images.githubusercontent.com/7287218/45205722-96d73d80-b2a0-11e8-82b1-07eb5482e556.png)

**In tab and mobile screens hamburger icon displays, and by clicking on that menu appears, and once you click on a menu item again it collapses. That is how the behavior is designed.** @anumehaagrawal this is the designed behavior right?

**Comment by tomzacchia** [comment]
I agree with you that on mobile screens, this bug is not visible perhaps due to the different browser. However on desktops, with Chrome, when the browser size is smaller than 1200px the nav links are permanently displayed. This might also be the case for android tablets, not iPads because the browser is also safari. 

My suggestion would be to use some Material components, which come with the dropdown functionality rather than re-designing one from scratch that is compatible across multiple browsers.

**Comment by Rupinderwadali** [comment]
@tomzacchia I recreate the issue. Yeah! there is a bug. Infact it is showing errors while scrolling as well.

**Comment by Tharangi** [comment]
@tomzacchia are you up to fix this bug?

**Comment by anumehaagrawal** [comment]
@tomzacchia  Should we open this bug fix to anyone ?

**Comment by sarnava1** [comment]
Can I try solving this @anumehaagrawal?
